examples/angular2/app.ts

```diff
@@ -0,0 +1,86 @@
+/// <reference path="typings/angular2/angular2.d.ts" />
+const ESC_KEY = 27;
+const ENTER_KEY = 13;
+
+import {Component, View, bootstrap, For, If} from 'angular2/angular2';
+import {TodoStore, Todo} from 'services/store';
```

Comment:
Comments, doesn't count. To be specific. Imports needs to be the first code in the file.